### PR TITLE
Fix some markdown links

### PR DESCRIPTION
**Pull request by Jolg42**
@nikolasburk 

I used remark for finding the broken links, like this:
- Install `npm install --global remark-cli remark-validate-links`
- Run in directory `remark -u validate-links .`

And still 3 to fix after this PR.

> README.md
    43:5-43:38  warning  Link to unknown file: `vscode-extension`                                                                                                                                         missing-file     remark-validate-links

> cli/README.md
    72:5-72:44  warning  Link to unknown file: `../introspection`                                                                                                                                         missing-file     remark-validate-links
  100:5-100:30  warning  Link to unknown file: `../photon`. Did you mean `../photonjs`                                                                                                                    missing-file     remark-validate-links

### CODE REVIEW

**Comment by Jolg42** [comment]
It would be good to do that in prisma2 as well I think, I found some broken links there as well :)

File: README.md [hunk 1]
```diff
@@ -36,7 +36,7 @@ Prisma Framework consists of several tools:
 - Tools
   - [CLI](cli) - setup and introspection
   - [Studio](studio) - GUI
-  - [Photon](photon) - data access
```
Comment (janpio) [inline README.md]: This is a reminder this page needs to be created.

File: README.md [hunk 2]
```diff
@@ -46,7 +46,7 @@ Prisma Framework consists of several tools:
 
 It also has several internal parts:
 
-- [SDK](sdk)
```
Comment (janpio) [inline README.md]: This is a reminder this page needs to be created.

**Comment by Jolg42** [comment]
@janpio Done :)

**Pull request merged**